doc/lsst.pipe.base/command-line-task-data-repo-howto.rst

```diff
@@ -0,0 +1,136 @@
+.. _command-line-task-data-repo-howto:
+
+#################################################################
+Using Butler data repositories and reruns with command-line tasks
+#################################################################
+
+:ref:`Command-line tasks <using-command-line-tasks>` use Butler data repositories for reading and writing data.
+Butler data repositories are specially-configured directories on a filesystem.
```

Comment:
I'm not aware of specific tickets at this time; @n8pease would know.  A trivial subsetting utility that works (terribly inefficiently) like a `CmdLineTask` that just copies input datasets to its output can be written in a few lines.  Relocation (without copying) and removal would require a bit more work.